### PR TITLE
Wording like license vs account and minor updates

### PR DESCRIPTION
**Pull request by marcoscheel**
* Change "disable License" to "disable account" as a license change as document won't impact the system and a license can not be disabled! But a user account can be disabled.
* Add the five org wide teams limit to the list of things that will prevent the option to show for global admins
* Added to note section the fact any change by the system will show up in the main channel and show as ne activity (bold)
* Emphasise the fact you have to remove member through the Teams UI/client, because the people often responsible are used to other tools like PowerShell, Azure Portal or Admin O365

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/create-an-org-wide-team.md [hunk 1]
```diff
@@ -35,11 +35,12 @@ These types of accounts won't be added to your org-wide team:
...
+As your organization's directory is updated to include new active users or if users no longer work at your company and their account is disabled, changes are automatically synced and the users are added or removed from the team. Team members can't leave an org-wide team. As a team owner, you can manually add or remove users if needed.
 
 > [!NOTE]
-> - If you don't see the **Org-wide** option when creating a team and you're a global admin, the feature might still be rolling out or your organization might have more than the current size limit of 5,000 members. We're looking to increase this limit in the future.
+> - If you don't see the **Org-wide** option when creating a team and you're a global admin, the feature might still be rolling out or you have reached the five **Org-wide** Teams limit or your organization might have more than the current size limit of 5,000 members. We're looking to increase this limit in the future.
```
Comment (JohanFreelancer9) [inline Teams/create-an-org-wide-team.md]:
```suggestion
> - If you don't see the **Org-wide** option when creating a team and you're a global admin, the feature might still be rolling out, you have reached the five **Org-wide** Teams limit, or your organization might have more than the current size limit of 5,000 members. We're looking to increase this limit in the future.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LanaChin** [comment]
Reached out to PM to review changes.

**Pull request merged**